frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -1017,350 +1065,447 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
+                                                        description: 'Single-user apps (mobile, desktop, embedded)',
+                                                    },
+                                                    {
+                                                        value: FeatureFlagEvaluationRuntime.SERVER,
+                                                        icon: <IconServer />,
+                                                        title: 'Server-side only',
+                                                        description: 'Multi-user systems in trusted environments',
+                                                    },
+                                                ].map((option) => (
+                                                    <div
+                                                        key={option.value}
+                                                        className={`border rounded-lg p-4 cursor-pointer transition-all hover:border-primary-light ${
+                                                            value === option.value
+                                                                ? 'border-primary bg-primary-highlight'
+                                                                : 'border-border'
+                                                        }`}
+                                                        onClick={() => onChange(option.value)}
+                                                    >
+                                                        <div className="flex items-start gap-3">
+                                                            <div className="text-lg text-muted">{option.icon}</div>
+                                                            <div className="flex-1">
+                                                                <div className="font-medium text-sm">
+                                                                    {option.title}
+                                                                </div>
+                                                                <div className="text-xs text-muted mt-1">
+                                                                    {option.description}
+                                                                </div>
+                                                            </div>
+                                                            <input
+                                                                type="radio"
+                                                                name="evaluation-environment"
+                                                                checked={value === option.value}
+                                                                onChange={() => onChange(option.value)}
+                                                                className="cursor-pointer"
+                                                            />
```

Comment:
Should we be using the `<LemonRadio />` component here?

Reply:
because `<LemonRadio />` is an opinionated component isn't really designed for this use case – it has a specific layout and doesn't really support much styling.  I wanted radio buttons that fit with this card design (which I thought looked nicer), and  I only really needed them to provide a semantic selection state instead of some whole involved component.